wallets/provider-tron-link/src/index.ts

```diff
@@ -23,27 +30,63 @@ export const config = {
   defaultNetwork: Networks.TRON,
```

Comment:
Default network is `tron`, When you are on `Ethereum` network on TronLink, it goes through an error. How we can fix it?